### DISCUSSION TITLE
what is the pods about production strategy?

### DISCUSSION THREAD

**Discussion by janrs-io** [jaegertracing/jaeger-operator#2183] [Q&A]
i install the operator by the command 
```kubectl create namespace observability # <1>
kubectl create -f https://github.com/jaegertracing/jaeger-operator/releases/download/v1.41.0/jaeger-operator.yaml -n observability # <2>
```
and create instance 
```apiVersion: jaegertracing.io/v1
kind: Jaeger
metadata:
  name: simple-prod
spec:
  strategy: production
  collector:
    maxReplicas: 5
    resources:
      limits:
        cpu: 100m
        memory: 128Mi
```

**the pods i got is just one pod and it is use `jaegertracing/all-in-one:1.42.0` mirror**

the pods shows like this
```root@k8s-master01:/home/john# kgpon jaeger-instance
NAME                     READY   STATUS    RESTARTS   AGE
jaeger-64b54fb6f-lb269   1/1     Running   0          9m47s
```

I'm wondering what the pods should look like, with the production policy in place, showing up correctly.
Can someone help me?

**Comment by desert3agle** [comment]
you need to specify the storage. once you do that, you'll have two pods, one with collector and other with agent and query.

**Reply by janrs-io** [reply]
"you need to specify the storage." does this means that i need to use es or loki , something like this?
